flow/scripts/util.tcl

```diff
@@ -31,6 +31,7 @@ proc repair_timing_helper { {hold_margin 1} } {
   if {$hold_margin || $::env(HOLD_SLACK_MARGIN) < 0} {
     append_env_var additional_args HOLD_SLACK_MARGIN -hold_margin 1
   }
+  append_env_var additional_args SETUP_MOVE_SEQUENCE -sequence 1
```

Comment:
Needs to be documented

Reply:
Is this done in the `flow/scripts/variables.yaml`?

Reply:
@mguthaus yes, and one needs to run `python3 flow/scripts/generate-variables-docs.py` to update the generated markdown

Reply:
I thought it best to put "repair" into the name. One other idea: `SETUP_REPAIR_SEQUENCE`